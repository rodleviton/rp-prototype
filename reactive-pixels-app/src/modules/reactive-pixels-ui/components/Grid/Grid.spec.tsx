import * as React from 'react';
import { render } from 'react-testing-library';
import WithTheme from '../../utils/test/WithTheme';
import { Grid } from './';

describe('<Grid />', () => {
  it('should render <Grid /> component', () => {
    const { container } = render(
      <WithTheme>
        <Grid />
      </WithTheme>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
