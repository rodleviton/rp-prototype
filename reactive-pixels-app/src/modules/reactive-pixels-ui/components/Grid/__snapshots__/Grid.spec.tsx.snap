// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Grid /> should render <Grid /> component 1`] = `
<div
  class="css-0 sc-bwzfXH hKiLMS sc-bdVaJa bDWFJH"
/>
`;
